frappe/core/doctype/rq_job/rq_job.py

```diff
@@ -161,7 +161,7 @@ def serialize_job(job: Job) -> frappe._dict:
 		started_at=convert_utc_to_system_timezone(job.started_at) if job.started_at else "",
 		ended_at=convert_utc_to_system_timezone(job.ended_at) if job.ended_at else "",
 		time_taken=(job.ended_at - job.started_at).total_seconds() if job.ended_at else "",
-		exc_info=job.exc_info,
+		exc_info=job.latest_result().exc_string,
```

Comment:
latest_result can be None for queued jobs?

Reply:
Yep, adding a check